src/main/java/nl/tudelft/ti2206/group9/gui/ExternalTicker.java

```diff
@@ -32,6 +32,10 @@ private void renderScene() {
 		GameWindow.addWorld(entities);
 
 		GameWindow.addOverlay(new Text(0, 16, "Score: " + State.getScore()));
+		GameWindow.addOverlay(new Text(0, 32, "Distance: " 
+				+ State.getTrack().moduloDistance()));
```

Comment:
Maybe you could also move moduloDistance() to State? It looks a bit weird that public getDistance() is moved to State but this one isn't :P